packages/poi-preset-karma/index.js

```diff
@@ -76,6 +71,32 @@ module.exports = (options = {}) => {
 
       delete webpackConfig.entry
 
+      if (coverage) {
+        /* for general usage */
+        webpackConfig.module.rules = [
+          {
+            test: /\.(jsx?)$/,
+            exclude: /(node_modules|\.test\.jsx?)/,
+            enforce: 'post',
+            loader: 'istanbul-instrumenter-loader',
+            query: {
+              esModules: true
+            }
+          }
+        ].concat(webpackConfig.module.rules)
+        /* for vue (assumes vue-loader) */
+        webpackConfig.module.rules = webpackConfig.module.rules
+          .map(r => {
```

Comment:
@egoist is there a better way to modify the webpack config than mapping over its rules like this?